zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/authorization/MappingRecord.java

```diff
@@ -24,10 +24,8 @@ public MappingRecord() {
     declareProperty(mappingKeyProp).declareProperty(claimNameProp).declareProperty(claimValueProp);
   }
 
-  public void wrap(final MappingRecord record) {
-    mappingKeyProp.setValue(record.getMappingKey());
-    claimNameProp.setValue(record.getClaimName());
-    claimValueProp.setValue(record.getClaimValue());
+  public void copy(final MappingRecord record) {
+    copyFrom(record);
```

Comment:
:thought_balloon: This naming now conflicts with the _other_ `copy` methods that return a new copy of itself. Since _this_ `copy` is just an alias for `copyFrom` which has a clearer name, could we just delete this and similar methods and just use `copyFrom` directly?

Reply:
I'll jsut remove it, it's not even used 🤡

Reply:
(Same in other records of course)